pkg/apiserver/apiserver.go

```diff
@@ -105,6 +105,7 @@ func registerCubeAPI(cfg *Config) http.Handler {
 		k8sApiExtend.Any("/clusters/:cluster/namespaces/:namespace/:resourceType/:resourceName", resourcemanage.ExtendHandle)
 		k8sApiExtend.Any("/clusters/:cluster/namespaces/:namespace/:resourceType", resourcemanage.ExtendHandle)
 		k8sApiExtend.POST("/clusters/:cluster/yaml/deploy", yamldeploy.Deploy)
+		k8sApiExtend.GET("/clusters/:cluster/projects/:project/ingressDomainSuffix", resourcemanage.IngressDomainSuffix)
```

Comment:
url path confused that I thought getting a specified project ingressDomainSuffix from a specified cluster, but in fact, we just do combination with project and cluster. So, is query params better?

Reply:
change to use params